be/src/Project/Web/Buyer/Buyer.Web/Shared/GraphQlResponseModels/MainNavigationLinksGraphQlResponseModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Buyer.Web.Shared.DomainModels.MainNavigationLinks;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Buyer.Web.Shared.GraphQlResponseModels
+{
+    public class MainNavigationLinksGraphQlResponseModel
+    {
+        [JsonProperty("page")]
+        public MainNavigation Page { get; set; }
+    }
+
+    public class MainNavigation
+    {
+        [JsonProperty("data")]
+        public MainNavigationData Data { get; set; }
+    }
+
+    public class MainNavigationData
+    {
+        [JsonProperty("attributes")]
+        public MainNavigatioDataAttributes Attributes { get; set; }
+    }
+
+    public class MainNavigatioDataAttributes
+    {
+        [JsonProperty("mainNavigationLinks")]
+        public MainNavigationDataAttibutesMainNavigationLinks MainNavigationLinks { get; set; }
+    }
+
+    public class MainNavigationDataAttibutesMainNavigationLinks
+    {
+        [JsonProperty("links")]
+        public List<MainNavigationDataAttibutesMainNavigationLinksLink> Links {  get; set; }
+    }
+
+    public class MainNavigationDataAttibutesMainNavigationLinksLink
```

Comment:
**suggestion (non-blocking):** I think it wouldn't be a bad idea to correct the class name. LinksLink?